subsys/partition_manager/Kconfig

```diff
@@ -101,6 +101,47 @@ config PM_EXTERNAL_FLASH_BASE
 
 endif
 
+config BUILD_WITH_TFM
+	bool
+	select PM_SINGLE_IMAGE
+
+if BUILD_WITH_TFM
+
+config PM_PARTITION_SIZE_TFM_RAM
+	hex
+	default 0x16000 if SOC_NRF9160
+	default 0x40000 if SOC_NRF5340_CPUAPP
+	help
+	  Memory set aside for the TFM_RAM partition.
+	  The prompt has been removed since the value is dictated by TFM and
+	  cannot be changed.
+
+config PM_PARTITION_SIZE_BL2
```

Comment:
What if BL2 is is disabled: https://github.com/zephyrproject-rtos/zephyr/blob/master/modules/Kconfig.tfm#L96-L97

Should this setting have a dependency ?

Reply:
The nRF platforms don't really support that.

Reply:
Not sure about that - we should be able to build TFM without BL2, in general

Reply:
Maybe, but right now it fails on the signing code in Zephyr.

Reply:
Ok that's something we need to have documented.
@b-gent FYI